docs/source/articles/configure-headless-chrome.html.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Configure Headless Chrome
+created_at: 2020/01/23
+excerpt: How to configure headless Chrome
+---
+
+# Configure Headless Chrome
+
+Workarea system tests are run within a headless instance of Chrome, driven by Selenium.
+Since Workarea 3.5.0, you can customize this setup using the following [configuration](/articles/configuration.html) values:
+
+| Config | Description |
+| --- | --- |
+| [`headless_chrome_options`](https://github.com/workarea-commerce/workarea/blob/v3.5.3/core/lib/workarea/configuration.rb#L953) | Options hash passed to the Selenium driver |
+| [`headless_chrome_args`](https://github.com/workarea-commerce/workarea/blob/v3.5.3/core/lib/workarea/configuration.rb#L956-L962) | Array of command line arguments passed to the Chrome executable |
+
+Replace or mutate the default values from within your own initializer:
+
+```
+# <your_application_root>/config/initializers/headless_chrome.rb 
```

Comment:
Feels like this should be in `test/test_helper.rb` or something like `test/support/headless_chrome.rb` to indicate this exists for testing purposes only.